hashicorp/vault
# UI: show banner when batch token expired

Previously, when logging in with a batch type token the UI would suddenly stop working with a permissions error when the batch token expires. This PR allows the UI to know when the batch token will expire and show the token expired warning when it does. 

**Testing steps**
* Create a batch token with a short TTL `vault token create -type=batch -ttl=30s`
* Log into the GUI with the returned token
* Wait for the TTL amount of time
* See that the GUI shows the same token expired banner as when a service type token expires

**Before and after**
<img width="1726" alt="GUI before (left) and after (right)" src="https://github.com/hashicorp/vault/assets/82459713/4e32d4db-b74e-480f-a36d-1c537e87ae62">

![batch-expire](https://github.com/hashicorp/vault/assets/82459713/d3c7b26a-4361-4c32-b6a0-ab329d703d92)
